app/config.ts

```diff
@@ -1,12 +1,33 @@
+// tslint:disable-next-line
+import { BigNumber } from "bignumber.js";
+import { asEtherNumber } from "./web3/utils";
+
 function getEnvValue(obj: any, key: string) {
   if (obj[key] === undefined) {
     throw new Error(`${key} is not exists in .env file`);
   }
-
   return obj[key];
 }
 
-export const commitmentContractAdress: string = getEnvValue(
-  process.env,
-  "COMMITMENT_CONTRACT_ADDRESS"
-);
+export let envPayload: any;
```

Comment:
why it's named `envPayload`? Probably it's one of the less informative names that it could have :D